projects/app/src/global.css

```
@@ -243,6 +243,27 @@
     --color-bg: var(--color-bg-light);
     --color-bg-loud: var(--color-bg-loud-light);
   }
+
+  .pyly-mdx {
+  }
```

Biome (1.9.4)

[error] An empty block isn't allowed.

Consider removing the empty block or adding styles inside it.

(lint/suspicious/noEmptyBlock)